syncd/scripts/saidump.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -e
+
+function debug()
+{
+  /usr/bin/logger "$1"
+}
+
+save_saidump_by_rdb() {
```

Comment:
Add new line before { to keep the same for entire file

Reply:
Fix it.